### PR TITLE
Embed docs add generic frame class to example

### PR DESCRIPTION
**Pull request by Toinne**
Made some general improvements for readability.

Only fixed 1 mistake https://github.com/zendertv/zender-docs/pull/1/commits/f8764963950d963ea0b49b22cc4aa594af908741.

### CODE REVIEW

**Review by jagij — APPROVED** [review]
👍

File: EMBED.md [hunk 1]
```diff
@@ -14,18 +14,18 @@ This will show the player inside the iframe and will automatically resize.
 
 Both targetId and channelId will be provided to you on the creation of your Zender account. They can also be found through the Zender admin console.
 
-# Zender Lobby mechanism
-By default when no streams are public there will nothing to see.
+### Zender Lobby mechanism
+By default no streams are public and there will be nothing to see.
```
Comment (jagij) [inline EMBED.md]: This changes the meaning a bit: it's an admins choice whether any stream is public.
I'd go with:
When there is no lobby stream and no streams are public, there will be nothing to see (or show?).

Comment (Toinne) [reply EMBED.md]: Just also removed the by default since that suggested us describing the initial configuration

**Comment by jagij** [comment]
Ik ga 'm gewoon mergen.

**Pull request merged**